Exiled.Events/EventArgs/Scp939/LungingEventArgs.cs

```diff
@@ -13,28 +13,19 @@ namespace Exiled.Events.EventArgs.Scp939
     /// <summary>
     ///     Contains all information before SCP-939 uses its lunge ability.
     /// </summary>
-    public class LungingEventArgs : IPlayerEvent, IDeniableEvent
+    public class LungingEventArgs : IPlayerEvent
     {
         /// <summary>
         ///     Initializes a new instance of the <see cref="LungingEventArgs" /> class.
         /// </summary>
         /// <param name="player">
         ///     <inheritdoc cref="Player" />
         /// </param>
-        /// <param name="isAllowed">
-        ///     <inheritdoc cref="IsAllowed" />
-        /// </param>
-        public LungingEventArgs(ReferenceHub player, bool isAllowed = true)
+        public LungingEventArgs(Player player)
         {
-            Player = Player.Get(player);
-            IsAllowed = isAllowed;
+            Player = player;
         }
 
-        /// <summary>
-        ///     Gets or sets a value indicating whether or not SCP-939 can lunge.
-        /// </summary>
-        public bool IsAllowed { get; set; }
```

Comment:
Use Obsolete

Reply:
Do I need to assign it to the cctor too?